### ISSUE TITLE
Nanoflow!

### ISSUE THREAD

**Issue by tieniber** [mendix/CustomString#11]
This widget would be perfect nanoflow use case!

**Comment by JelteMX** [comment]
Agreed, but I need to discuss how we are handling backwards compatibility. The problem here is that if we introduce Nanoflows, we're breaking backwards compatibility with older versions (XML is not accepted) of Mendix

**Comment by mihaimiculescu** [comment]
4 widgets in the bundle?
(2 backward-compatible and 2 for new projects only)

**Comment by JelteMX** [comment]
@mihaimiculescu that might work

**Comment by Andries-Smit** [comment]
Just release a newer version 7.14 and up. Projects with older version will only access older version.
